generate/generate.go

```diff
@@ -709,12 +709,11 @@ func (g *Generator) ClearProcessCapabilities() {
 
 // AddProcessCapability adds a process capability into g.spec.Process.Capabilities.
 func (g *Generator) AddProcessCapability(c string) error {
-	if err := checkCap(c, g.HostSpecific); err != nil {
+	cp := fmt.Sprintf("CAP_%s", strings.ToUpper(c))
```

Comment:
Why are we adding the `CAP_` prefix here?  I'd rather require `AddProcessCapability` callers give us a valid string without massaging it internally.